### ISSUE TITLE
Delphi compatibility - Anonymous method, Attributes, and Inline variables support. 

### ISSUE THREAD

**Issue by AlekId** [newpascal/newpascal#6]
Is this project alive?
If yes, please add those Delphi-compatible features. For now, NewPascal is just outdated FPC fork,
**but** those features will make huge difference.
Since FPC just dropped their goal to be Delphi-compative it's perfect opportunity to make your work really important. And I think it's the **only** way.

**Comment by lhengen** [comment]
I agree that in order to maximize it's footprint, FPC needs to be completely Delphi compatible, or we need an alternate open source compiler.  The Lazarus IDE experience is not up to the same level as the Delphi IDE and in many ways VS.NET and other commercial IDEs make Delphi's look primitive.  The one advantage Lazarus/FPC has is the ability to compile 32 and 64 bit EXEs for different OSes using the same GUI widgets.  In my experience, that works very well.  FireMonkey is only supported on 2 OSes, only 1 of which can be targeted in 64 bit.  Between ARC/Classic, 32/64, and FireMonkey support for different OSes, the Delphi platform is fragmented, unable to offer the xplatform targets you have with C# now.
